travis.yml

```diff
@@ -0,0 +1,13 @@
+language: node_js
+node_js:
+    - "6.11.0"
+install:
+    - npm install
+script:
+    - npm test
+services:
+  - mongodb
+cache: 
+  - npm install
+notifications:
+  webhooks: https://coveralls.io/webhook?repo_token=AlQ0K4WyO8yno9TSEP9t3SoyrkmoYgI8I
```

Comment:
I suggest this repo_token should also be included in .env file as it is also a sensitive data